bin/fuel-core/src/cli/run/graphql.rs

```diff
@@ -66,6 +66,13 @@ pub struct GraphQLArgs {
     #[clap(long = "api-request-timeout", default_value = "30s", env)]
     pub api_request_timeout: humantime::Duration,
 
+    #[clap(
+        long = "required-fuel-block-height-tolerance",
```

Comment:
```suggestion
        long = "graphql_required-fuel-block-height-tolerance",
```

Reply:
Also it would be nice to have a comment to describe the functionality behind this CLI argument.